### PR TITLE
chore: add some missing ethers.js constants

### PR DESCRIPTION
**Pull request by xtools-at**
## PR-Codex overview
This PR focuses on adding two constants, `zeroHash` and `presignMessagePrefix`, and updating the `hashMessage` function to use the `presignMessagePrefix` constant.

### Detailed summary
- Added `zeroHash` constant in `constants/bytes.ts`
- Added `presignMessagePrefix` constant in `constants/strings.ts`
- Updated `hashMessage` function in `utils/signature/hashMessage.ts` to use `presignMessagePrefix` constant

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by xtools-at** [comment]
asked for in #447 and #667

File: src/constants/number.ts [hunk 1]
```diff
@@ -96,3 +96,11 @@ export const maxUint232 = 2n ** 232n - 1n
...
 export const maxUint248 = 2n ** 248n - 1n
 export const maxUint256 = 2n ** 256n - 1n
+
+export const weiPerEther = 1000000000000000000n
+export const zero = 0n
+export const negativeOne = -1n
+export const one = 1n
+export const two = 2n
+export const N =
+  0xfffffffffffffffffffffffffffffffebaaedce6af48a03bbfd25e8cd0364141n
```
Comment (jxom) [inline src/constants/number.ts:106]: I'm not too sure I see much value in these apart from `weiPerEther` which we already export in `src/constants/unit.ts`

File: src/constants/strings.ts [hunk 2]
```diff
@@ -0,0 +1,2 @@
+export const etherSymbol = 'Ξ'
```
Comment (jxom) [inline src/constants/strings.ts:1]: I'm not sure if we need to export this. Can't consumers just google "ether symbol"?

File: src/constants/strings.ts [hunk 3]
```diff
@@ -0,0 +1,2 @@
...
+export const messagePrefix = '\x19Ethereum Signed Message:\n'
```
Comment (jxom) [inline src/constants/strings.ts:2]: Can we import this in `src/utils/signature/hashMessage.ts`?

File: src/constants/bytes.ts [hunk 4]
```diff
@@ -0,0 +1,2 @@
+export const hashZero =
```
Comment (jxom) [inline src/constants/bytes.ts:1]:
```suggestion
export const zeroHash =
```

File: src/constants/strings.ts (hunk 3 above)
Comment (jxom) [inline src/constants/strings.ts:2]:
```suggestion
export const presignMessagePrefix = '\x19Ethereum Signed Message:\n'
```

**Review by jxom — CHANGES_REQUESTED** [review]
Think `zeroHash` and `messagePrefix` may be the useful ones here.

**Comment by xtools-at** [comment]
committed requested changes

**Pull request merged**